index.html

```diff
@@ -181,7 +182,7 @@ <h4>LET'S CONNECT</h4>
         <ul>
           <li><input type="text" maxlength="30" placeholder="Name" required></li>
           <li><input type="email" placeholder="Enter E-mail" required></li>
-          <li><textarea name="msg" placeholder="Write your message here" maxlength="500"></textarea></li>
+          <li><textarea name="msg" placeholder="Write your message here" maxlength="500" required></textarea></li>
           <li><input type="submit" value="Get It Touch"></li>
```

Comment:
- [ ] (OPTIONAL) It would be great if you could apply the correct button interactions for your button in order to improve the user interface
![Capture1](https://user-images.githubusercontent.com/85113567/154197954-d649ddea-4099-4339-a121-b81e84b834a0.PNG)
.